skl2onnx/operator_converters/text_vectoriser.py

```diff
@@ -209,6 +209,15 @@ def convert_sklearn_text_vectorizer(scope, operator, container):
             "https://github.com/onnx/sklearn-onnx/issues.")
 
     if op.lowercase or op.stop_words_:
+
```

Comment:
What happens when the above if condition is False? flatten variable would be undefined, won't it? And that's being used below.

Reply:
StringNormalizer is the only operator which does not accept 2D inputs (or inputs with more than one row). Flatten is not needed when this operator is not used.

Reply:
Check line 242, you are using flatten there(which would be undefined in the case I mentioned).